README.md

```diff
@@ -1,424 +1,159 @@
+Once you have a project set up and have enabled an API, you are ready to begin
+using a client library to call the API.
 
-### Cloud Spanner
+### Choosing a client library
 
-- `gem install google-cloud-spanner`
-- [google-cloud-spanner README](google-cloud-spanner/README.md)
-- [google-cloud-spanner API documentation](https://googleapis.dev/ruby/google-cloud-spanner/latest)
-- [google-cloud-spanner on RubyGems](https://rubygems.org/gems/google-cloud-spanner)
-- [Google Cloud Spanner documentation](https://cloud.google.com/spanner/docs)
+This repository contains two types of API client Rubygems: the _main library_
+for the API (e.g. the `google-cloud-translate` gem for the Translation service),
+and one ore more _versioned libraries_ for different versions of the service
+(e.g. `google-cloud-translate-v2` and `google-cloud-translate-v3` for versions
+2 and 3 of the service).
 
-### Cloud Speech-to-Text
+In most cases, you should install the _main library_ (the one without a service
+version in the name). This library will provide all the required code for making
+calls to the API. It may also provide additional convenience code implementing
+common client-side workflows or best practices. Often the main library will
+bring in one or more versioned libraries as dependencies, and the client and
+data type classes you will use may actually be defined in a versioned library,
+but installing the main library will ensure you have access to the best tools
+and interfaces for interacting with the service.
 
-- `gem install google-cloud-speech`
-- [google-cloud-speech README](google-cloud-speech/README.md)
-- [google-cloud-speech API documentation](https://googleapis.dev/ruby/google-cloud-speech/latest)
-- [google-cloud-speech on RubyGems](https://rubygems.org/gems/google-cloud-speech)
-- [Google Cloud Speech-to-Text documentation](https://cloud.google.com/speech/docs)
+The _versioned libraries_ are lower-level libraries that target a specific
+version of the service. You may choose to intall a versioned library directly,
+instead of or in addition to the main library, to handle advanced use cases that
+require lower level access.
+
+Note: Many services may also provide client libraries with names beginning with
+`google-apis-`. Those clients are developed in a different repository, and
+utilize an older client technology that lacks some of the performance and ease
+of use benefits of the clients in the google-cloud-ruby repository. The older
+clients may cover some services for which a google-cloud-ruby client is not yet
+available, but for services that are covered, we generally recommend the clients
+in the google-cloud-ruby repository over the older ones.
+
+Most client libraries have directories in this repository, or you can look up
+the name of the client library to use in the documentation for the service you
+are using. Install this library as a Ruby gem, or add it to your Gemfile.
```

Comment:
"rubygems", "Rubygems" or "Ruby gem" (with a space), which is correct?

Reply:
Changed everything to "RubyGem".